### ISSUE TITLE
Yabridge installer requires Wine 9.4  - no option to downgrade on Fedora 40

### ISSUE THREAD

**Issue by GitCrush** [robbert-vdh/yabridge#343]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

After upgrading my OS to Fedora 40, Wine has been updated to version 9.9 and there is no option to downgrade to version 9.4 , as required by yabridge installer 
[https://dl.winehq.org/wine-builds/fedora/40/x86_64/](url)

### What did you expect to happen?

Yabridge installer should not require version 9.4 as fixed dependency

### What actually happened?

`$ sudo dnf install yabridge --refresh
Problem: conflicting requests
  - nothing provides wine(x86-32) = 9.4 needed by yabridge-5.1.0-1.fc40.x86_64 from copr:copr.fedorainfracloud.org:patrickl:yabridge
  - nothing provides wine = 9.4 needed by yabridge-5.1.0-1.fc40.x86_64 from copr:copr.fedorainfracloud.org:patrickl:yabridge
`

### Operating system

Fedora 40

### How did you install yabridge?

COPR

### yabridge version

5.1.0-1.fc40 

### yabridgectl version

 5.1.0-1.fc40 

### Wine version

Wine staging 9.9

### Plugin

all

### Plugin type

VST2

### Plugin architecture

64-bit

### Host

Bitwig Studio 4.5

**Comment by a118n** [comment]
Agreed. Same problem.

**Comment by Cobesz** [comment]
Same here, also tried adding the fedora 39 repo. Didn't work

**Comment by jo5huajohn** [comment]
I'm facing the same issue as well when trying to install it on fedora. I'm running Fedora Silverblue 40 with winehq-staging-1:9.10-1.1.x86_64 from the wine repo.

**Comment by sleeplessKomodo** [comment]
Yeah, I'm confused. 

              With the merge of https://gitlab.winehq.org/wine/wine/-/merge_requests/5400 this issue is now resolved again for Wine 9.8 onwards.

_Originally posted by @robbert-vdh in https://github.com/robbert-vdh/yabridge/issues/320#issuecomment-2094218649_

yet 

```
Error:
 Problem: conflicting requests
  - nothing provides wine(x86-32) = 9.4 needed by yabridge-5.1.0-1.fc40.x86_64 from copr:copr.fedorainfracloud.org:patrickl:yabridge
```

**Comment by jo5huajohn** [comment]
It seems to be an issue on the repo side. Downloading the binary from releases seems to bypass the issue. Maybe some dependency config?

**Comment by sleeplessKomodo** [comment]
Yep, using the prebuilt binary does bypass the issue. I'm not sure where to report this.
